patroni/postgresql.py

```diff
@@ -1547,14 +1544,16 @@ def clone(self, clone_member):
 
     def bootstrap(self, config):
         """ Initialize a new node from scratch and start it. """
+        pg_hba = config.get('pg_hba', [])
         method = config.get('method') or 'initdb'
         self._running_custom_bootstrap = method != 'initdb' and method in config and 'command' in config[method]
         if self._running_custom_bootstrap:
             do_initialize = self._custom_bootstrap
             config = config[method]
         else:
             do_initialize = self._initdb
-        return do_initialize(config) and self._configure_server_parameters() and self.start()
+        return do_initialize(config) and self.write_pg_hba(pg_hba) and self.save_configuration_files() \
```

Comment:
Would it makes sense to look for pg_hba.conf from the config[method] at this point (allowing  to configure per-bootstrap pg_hba.conf) and falling back to the 'generic' pg_hba.conf in the bootstrap only if the method-specific one is not defined?

Reply:
I don't really want to implement anything new here, just want to fix a bug.
Taking into account that we can't use more than one custom bootstrap method it's not really worth it to make it possible to allow it to be configurable per section.